src/proposals/20251002/SparkEthereum_20251002.sol

```diff
@@ -0,0 +1,292 @@
+    function setNetworkLimit(bytes32 subnetwork, uint256 amount) external;
+    function setOperatorNetworkShares(bytes32 subnetwork, address operator, uint256 shares) external;
+    function setHook(address hook) external;
+    function OPERATOR_VAULT_OPT_IN_SERVICE() external view returns (address);
+    function OPERATOR_NETWORK_OPT_IN_SERVICE() external view returns (address);
+    function OPERATOR_NETWORK_SHARES_SET_ROLE() external view returns (bytes32);
+    function grantRole(bytes32 role, address account) external;
+}
+
+interface IOperatorRegistry {
+    function registerOperator() external;
+}
+
+interface IOptInService {
+    function optIn(address where) external;
+}
+
+interface ISparkVaultV2 {
+    function asset() external view returns (address);
+    function grantRole(bytes32 role, address account) external;
+    function setDepositCap(uint256 newCap) external;
+    function SETTER_ROLE() external view returns (bytes32);
+    function setVsrBounds(uint256 minVsr_, uint256 maxVsr_) external;
+    function TAKER_ROLE() external view returns (bytes32);
+}
+
+interface IVetoSlasher {
+    function setResolver(uint96 identifier, address resolver, bytes calldata hints) external;
+}
+
+/**
+ * @title  October 02, 2025 Spark Ethereum Proposal
+ * @notice Spark USDS Morpho Vault:
+ *         - Increase PT-USDe-27Nov Supply Cap
+ *         Spark Savings:
+ *         - Launch Savings v2 Vaults for USDC, USDT, and ETH
+ *         SparkLend:
+ *         - Increase LBTC Supply Cap Automator Parameters
+ *         - Reduce Stablecoin Market Reserve Factors
+ *         - Claim Accrued Reserves for USDS and DAI
+ *         Spark Liquidity Layer:
+ *         - Onboard SparkLend ETH
+ *         - Claim Aave Core aUSDS Rewards
+ *         - Add transferAsset Rate Limit for SYRUP
+ *         Spark Treasury:
+ *         - Transfer Share of Ethena Direct Allocation Net Profit to Grove
+ *         - Spark Foundation Grant
+ *         SPK Staking:
+ *         - Configure Symbiotic Instance
+ * @author Phoenix Labs
+ * Forum:  https://forum.sky.money/t/october-2-2025-proposed-changes-to-spark-for-upcoming-spell/27191
+ * Vote:   https://vote.sky.money/polling/QmUn84ag
+ *         https://vote.sky.money/polling/QmSaMJWy
+ *         https://vote.sky.money/polling/QmdY24Cm
+ *         https://vote.sky.money/polling/QmREvn1i
+ *         https://vote.sky.money/polling/QmerdKkX
+ *         https://vote.sky.money/polling/QmXYRjmQ
+ *         https://vote.sky.money/polling/QmeKTbg6
+ */
+contract SparkEthereum_20251002 is SparkPayloadEthereum {
+
+    // > bc -l <<< 'scale=27; e( l(1.1)/(60 * 60 * 24 * 365) )'
+    //   1.000000003022265980097387650
+    uint256 internal constant TEN_PCT_APY = 1.000000003022265980097387650e27;
+
+    // > bc -l <<< 'scale=27; e( l(1.05)/(60 * 60 * 24 * 365) )'
+    //   1.000000001547125957863212448
+    uint256 internal constant FIVE_PCT_APY = 1.000000001547125957863212448e27;
```

Comment:
These are currently in reverse order from the test file, maybe also put FIVE first? 🙂

Reply:
reordered